packagezip.sh

```diff
@@ -19,14 +19,18 @@ cd $name
 cp ../target/*database.zip .
 ln -s ../tomcat tomcat
 cp ../*.png .
-cp ../openmrs-standalone-runtime.properties .
+cp ../openmrs-runtime.properties .
 cp ../target/*.jar .
-cp ../target/openmrs-standalone.war tomcat/webapps/openmrs-standalone.war
```

Comment:
Is there a reason why you renamed this?

Reply:
yes this was coping `../target/openmrs-standalone.war` yet in target we have `../target/openmrs.war` also the in the standalone code we look for `openmrs.war` in the webapps directory not `openmrs-standalone.war`

Reply:
What did we do to change it from the original name?